core/core-progress-bar.el

```diff
@@ -33,7 +33,11 @@
 (defvar spacemacs-loading-dots-chunk-threshold 0)
 
 (defun spacemacs/init-progress-bar (max)
-  "Initialize the progress bar."
+  "Conditionally initialize the package-loading progress bar.
+
+When the customziation variable `dotspacemacs-loading-progress-bar' is non-nil,
```

Comment:
Can you just do what a said?

Reply:
Do I still need to do anything after your new commits? There are still new lines indicating the no-op when the variable is `nil`.

At the moment my Emacs is crashing because I listened to the warning coming from Magit #15992 .